src/add-patient-link.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import singleSpaReact from "single-spa-react";
+import { navigateToUrl } from "single-spa";
+import { Add20 } from "@carbon/icons-react";
+export default singleSpaReact({
```

Comment:
Don't use this. We have an abstraction.

Maybe please start by reading (and understanding!) the existing code.